src/util/transfer.ts

```diff
@@ -167,31 +166,45 @@ export const createTransaction = async (recipient :string, sendXPX :string, mess
  * @param message
  * @memberof ViewTransferComponent
  */
+//random account info
+const test_address = "VAIHSS-J72IDB-S5FGEB-G5JHV6-6PNG3T-YDWMZV-FA27" 
+const test_publicKey = "18B87A14FD660B6F8AFCA0430BB9C668B15B7B14162C78048248B941F7ED93CE"
```

Comment:
no hard code of sample account, please use all 0 public key then get the address from it